File: CS/SchedulerGSync/SchedulerGSync/DAL/SchedulerGSyncDbContext.cs
using Microsoft.EntityFrameworkCore;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SchedulerGSync.DAL {
    public class SchedulerGSyncDbContext : DbContext {
        public static readonly string DBFileName = $"{nameof(SchedulerGSync)}.sqlite";

        public DbSet<AppointmentObject> AppointmentObjects { get; set; }

        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder) {
            optionsBuilder.UseSqlite($"Filename={DBFileName}");
        }
    }
}

File: VB/SchedulerGSync/SchedulerGSync/DAL/SchedulerGSyncDbContext.vb
Imports Microsoft.EntityFrameworkCore
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks

Namespace SchedulerGSync.DAL
	Public Class SchedulerGSyncDbContext
		Inherits DbContext

		Public Shared ReadOnly DBFileName As String = $"{NameOf(SchedulerGSync)}.sqlite"

		Public Property AppointmentObjects() As DbSet(Of AppointmentObject)

		Protected Overrides Sub OnConfiguring(ByVal optionsBuilder As DbContextOptionsBuilder)
			optionsBuilder.UseSqlite($"Filename={DBFileName}")
		End Sub
	End Class
End Namespace
